src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java

```diff
@@ -201,7 +202,8 @@ else if (lobClass == Blob.class)
                         if (!verified) {
                             // Odd CharacterStream length will throw this exception
                             if (!e.getMessage().contains(TestResource.getResource("R_badStreamLength"))) {
-                                assertTrue(e.getMessage().contains(TestResource.getResource("R_streamReadError")));
+                                assertTrue(e.getMessage().contains(TestResource.getResource("R_streamReadError"))
+                                        || e.getMessage().contains(TestResource.getResource("R_aeStreamReadError")));
```

Comment:
should we check if AE is enabled to see which error it should be?

Reply:
The only really good way is to check the connection string, and that's honestly pretty hacky and I don't want to add the extra logic when the test doesn't really care if AE is on or off.

Reply:
yeah I did that in PR 1371, we should agree to do the same thing, if we're not going to check then I'll remove that from that PR too

Reply:
I'm ok with not checking, it doesn't have to be consistent persay.

Reply:
use ds.getColumnEncryptionSetting() as per discussion